lci_strategic_plugin/config/parameters.yaml

```diff
@@ -35,6 +35,20 @@ desired_distance_to_stop_buffer : 15.0
 # Double: The minimum period in seconds which a maneuver plan must cover if the plugin wishes to control the whole plan
 min_maneuver_planning_period : 15.1
 
+# Double: Update time interval of carma streets. Units in s
+carma_streets_update_interval : 1.0
+
+# Double: Minimum vehicle reaction time. Units in s
+reaction_time : 2.0
+
+# Bool: Switch case for usecase 2 and usecase 3; 0 for UC2 and 1 for UC3
+case_switch : 0
```

Comment:
See my naming comment below for config.h. 
Also the comment here needs to be updated based on that.